src/celeritas/phys/PhysicsParams.hh

```diff
@@ -34,14 +34,83 @@ class ParticleParams;
 
 //---------------------------------------------------------------------------//
 /*!
- * Physics configuration options.
+ * Particle-dependent physics configuration options.
+ *
+ * These parameters have different values for electrons/positrons and
```

Comment:
Maybe also explicit here the convention:
electrons/positrons (light) and muons/hadrons (heavy). Or the other way around: heavy particles (e-/e+)...

Since it is kind of arbitrary, repeating in the documentation helps tracking the definition while navigating the code.